cypress/e2e/room_board/shareBoardCardLinkInRoom.feature

```diff
@@ -7,7 +9,7 @@ Feature: Room Board - To share a board card link
         Given I am logged in as a '<teacher_2>' at '<namespace>'
         Given I am logged in as a '<teacher_1>' at '<namespace>'
 
-        # pre-condition: first teacher creates a room
+        # pre-condition: first teacher creates a room with second teacher assigned
```

Comment:
same as above

Reply:
changed this phrase back the way it is now.